slack/package.yaml

```diff
@@ -24,16 +24,18 @@ pipelines:
     description: |
       Forwards all events arriving on the topic `slack` to Slack.
     definition: |
+      // tql2
       // Slack requires every message to arrive in a JSON body in the format
       // `{text: "<message>"}`. To achieve that, we just write the content out
       // line by line and then read it back in, which gives us a human-readable
       // dump of the events in Slack, no matter what form they were in before.
-      subscribe slack
-      | write lines
-      | read lines
-      | put text=line
-      | batch 1
-      | to http POST "{{ inputs.webhook-url }}" write json
+      subscribe "slack"
+      write_lines
+      read_lines
+      select text=line
+      batch 1
+      write_json
+      save_http "{{ inputs.webhook-url }}", method="POST"
```

Comment:
You could use `to` here.

Reply:
I've changed this but I'm not sure this is a win. When reading the pipeline, the `write_json` now needs to specified out of line, after the URL.

Reply:
IIRC it should work without `write_json`, as HTTP has JSON as its default format.